tests/test_celery_unique.py

```diff
@@ -331,7 +327,7 @@ def test_creates_new_task_record_when_extant_task_exists(self):
         )
         self.assertIsInstance(rs, AsyncResult)
         self.assertNotEqual(self.redis_client.get(self.test_unique_redis_key), test_extant_tracked_task_id)
-        self.assertEqual(self.redis_client.get(self.test_unique_redis_key), rs.task_id)
+        self.assertEqual(self.redis_client.get(self.test_unique_redis_key), rs.task_id.encode())
```

Comment:
Sorry, when I was suggesting simply using `celery.uuid`, I was thinking that would also allow us to drop the `.encode()` part as well.  Is using `str.encode` strictly necessary here, or can we simply use `uuid()` and `rs.task_id` throughout this test suite?